docs/reference/modules/ROOT/pages/queries.adoc

```diff
@@ -218,7 +236,7 @@ Crux supports a handful of custom https://edn-query-language.org/eql/1.0.0/speci
 For full details on what's supported in the projection-spec, see the https://edn-query-language.org/eql/1.0.0/specification.html[EQL specification^]
 
 [#return-maps]
-== Returning maps
+=== Returning maps
```

Comment:
Note this was its own section previously because `:keys` is a sibling of `:find` - I don't have strong opinions against folding it under Find though